network-api/networkapi/campaign/views.py

```diff
@@ -172,13 +172,16 @@ def petition_submission(request, petition):
         )
 
     data = {
+        "json": True,
```

Comment:
Since "json" will always = True and "event_type" will always = "crm_petition_data",
I moved these two lines into the "data" object, that way when we send the message to SQS it will be all one flattened object like requested.

Reply:
except now you've changed the object that gets sent. Note that the original code sends a payload fo the form:
```
{
  app: settings.HEROKU_APP_NAME,
  timestamp: datetime.now().isoformat(),
  data: {
    json: true,
    form: {...lots of key/value pairs...},
    event_type: 'crm_petition_data'
  }
}
```
That is, it's an object with three keys, that inform Basket what format the `form` property has (it will be a JSON serialized object), and which event type this is.

With your change, we would now be sending:
```
{
  app: settings.HEROKU_APP_NAME,
  timestamp: datetime.now().isoformat(),
  data: {
    json: true,
    /* form is no longer defined */
    event_type: 'crm_petition_data',
   ...lots of key/value pairs...
  }
}
```